### DISCUSSION TITLE
Error Nuevo Enunciado

### DISCUSSION THREAD

**Discussion by sbarriag** [IIC2133-PUC/2024-2#110] [Tarea 1] [answered]
Hola, es mi idea o se equivocaron en subir el nuevo enunciado, sale lo siguiente 
![image](https://github.com/user-attachments/assets/6188bb83-2c60-46f3-804b-523ffd8792a0)
Como que no salen ni Happiness Buff ni Festival 
Muchas gracias

**Comment by 4n1t4** [comment, marked as answer]
Arreglado, gracias
